Application/EdFi.Ods.AdminApi/Infrastructure/Services/ClaimSetEditor/GetResourcesByClaimSetIdQueryV53Service.cs

```diff
@@ -42,7 +40,7 @@ internal static void AddChildResourcesToParents(IReadOnlyList<ResourceClaim> chi
             }
         }
 
-        internal IList<ResourceClaim> GetParentResources(int claimSetId)
+        internal IList<ResourceClaim> GetParentResources(int claimSetId, bool IsFiveThreeCqe)

```

Comment:
I'm not strongly concerned, but I am a little curious why this change is being made now in a patch release that is dedicated to fixing the System.Data.SqlClient vulnerability?

Is this something we overlooked in the previous cqe-related pull request?

Reply:
ReadChangesActionSupported flag was set by reading the actions from securitContext on the constructor. This did not cause error on the previous implementation. After the "Security Compatibility" package update. Causing the error on the constructor.